<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TT_GroupBoxImportGsi</class>
 <widget class="QGroupBox" name="TT_GroupBoxImportGsi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GroupBox</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="fr_file">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLineEdit" name="le_file">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string>Select a GSI file</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tb_open">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset theme="QIcon::ThemeIcon::DocumentOpen"/>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="lw_preview">
     <property name="font">
      <font>
       <family>Monospace</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SelectionMode::NoSelection</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gb_points">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QTableWidget" name="tw_points">
        <property name="font">
         <font>
          <family>Monospace</family>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SelectionMode::NoSelection</enum>
        </property>
        <property name="horizontalScrollMode">
         <enum>QAbstractItemView::ScrollMode::ScrollPerPixel</enum>
        </property>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>70</number>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Type</string>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
           <bold>false</bold>
          </font>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Name</string>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Parameters</string>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lb_nbPoints">
        <property name="text">
         <string>Those 0 points will be added</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>tb_open</sender>
   <signal>pressed()</signal>
   <receiver>TT_GroupBoxImportGsi</receiver>
   <slot>slot_tbOpenPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>553</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>290</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slot_tbOpenPressed()</slot>
 </slots>
</ui>
